components/script/dom/webgl_extensions/extensions.rs

```diff
@@ -52,7 +64,9 @@ impl Default for WebGLExtensionFeatures {
             disabled_tex_types: DEFAULT_DISABLED_TEX_TYPES.iter().cloned().collect(),
             not_filterable_tex_types: DEFAULT_NOT_FILTERABLE_TEX_TYPES.iter().cloned().collect(),
             effective_tex_internal_formats: HashMap::new(),
-            query_parameter_handlers: HashMap::new()
+            query_parameter_handlers: HashMap::new(),
+            hint_targets: HashSet::new(),
+            disabled_get_parameter_names: DEFAULT_DISABLED_GET_PARAMETER_NAMES.iter().cloned().collect(),
```

Comment:
If a lot WebGLContext are created they will exhaust GPU memory before the HashMap heap being noticeable. But from your experience in Stylo, is there any alternative, such a SmallVec? I expect that more extensions will use the same HashMap.